source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,570 @@
+
+----
+
+###################################
+Mobile Deployment via Custom Builds
+###################################
+
+Choosing this model means you’ve decided not to use the mobile applications Mattermost has made available via the public app stores. 
+
+This also means you will need to maintain these applications. Maintenance includes rebuilding and incorporating feature and/or security updates. Otherwise, your applications will not match the functionality of our publicly available applications, and could be incompatible with future versions of the Mattermost server.
+
+Finally, this choice also means you cannot use the hosted version of the Mattermost Push Notification Service (HPNS).
+
+A great first step for this deployment model involves `reviewing the documentation <https://developers.mattermost.com/contribute/mobile/>`_. Your development team should run through the requirements for building the applications. This will help understand the overall impact when choosing this option. This documentation provides guidance on building, compiling, and signing. It also includes information for customizing the apps.
+
+We’ve provided some recommended sections below.
+
+- Developer Setup
+- Build Your Own App
+- Push Notifications
+
+*********************************
+Distribution Through an App Store
+*********************************
+
+Once you have built your own apps you will need to distribute them. There are two options:
+
+Option 1 - Enterprise App Store via EMM Provider (most common and recommended): 
+
+- This is the most common way for customers to distribute their apps
+- Once added to your own "Enterprise App Store", user can download
+
+Option 2 - Apple App Store & Google Play (less common):
+
+- To submit an app to the official app stores, you need to submit the app to Apple/Google for review
+- This is the same process Mattermost uses make the apps available for everyone.
+- This process is more common if you are looking to white label the app to remove Mattermost branding
+- This is not the typical approach
+
+******************************************************
+Using an EMM Provider with Your Custom Mattermost Apps
+******************************************************
+
+If you will be using an EMM provider, note that Mattermost does not support App Wrapping. Instead we use the AppConfig standard.
+
+In some instances, there is an incompatibility with App Wrapping and React Native Applications. React Native is the technology used to develop the Mattermost mobile applications. The Mattermost mobile app will not function properly when using app wrapping.
+
+Though app wrapping was common practice, the AppConfig standard is replacing this approach. App wrapping is still often an option during the EMM configuration. Again, AppConfig is the modern standard used by Mattermost. It is the only supported method for securing Mattermost mobile applications via an EMM provider.
+
```

Comment:
Could both paragraphs above about app wrapping go in the primer section, and then we just need emphasise the use of AppConfig here. Coudl we just add a one liner Note here to state AppWrapping is not supported.

Reply:
For most content, no one read “top to bottom” or “front to back”. So repetition is by design. In this case, I think we can leave it. 2/5

Reply:
0/5 where this goes, but now that I read it again I think we should mention that the majority or the app wrappers provided by EMMs do not support WebSocket connections.

Reply:
+1 Fine with adding that.

Reply:
5/5 reads as desired.